tools/projective_transformation/projective_transformation.py

```diff
@@ -36,26 +47,65 @@ def warp_coords_batch(coord_map, shape, dtype=np.float64, batch_size=1000000):
     return coords
 
 
-def transform(moving_image, fixed_image, warp_matrix, out):
-    moving_image = skimage.io.imread(moving_image)
-    fixed_image = skimage.io.imread(fixed_image)
-    warp_matrix = pd.read_csv(warp_matrix, delimiter="\t", header=None)
-    warp_matrix = np.array(warp_matrix)
+def transform(moving_fn, fixed_fn, warp_mat, out):
 
-    trans = ProjectiveTransform(matrix=warp_matrix)
-    warped_coords = warp_coords_batch(trans, fixed_image.shape)
-    t = map_coordinates(moving_image, warped_coords, mode='reflect')
+    moving = skimage.io.imread(moving_fn)
+    extension = os.path.splitext(moving_fn)[1]
+    nDims = len(moving.shape)
+    assert nDims in [2, 3, 4, 5, 6], 'only support to 6D dimensions'
```

Comment:
"We only support up to 6 dimensions"

Reply:
Thanks for your prompt response. 

But the testing seemed failed. I had no idea.